crates/task-impls/src/view_sync.rs

```diff
@@ -782,6 +787,13 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                     return (None, self);
                 }
 
+                // cancel poll for votes
+                self.network
```

Comment:
Same comment we want `CancelPollForViewSyncVotes`

Reply:
Also I think we should `CancelPollForViewSyncCertificate` here as well

Reply:
You're right 🙈

Reply:
Should we also cancel the certificate poll in the case of a timeout?

Reply:
Which certificate and which timeout?

Reply:
In the case of a `Timeout` event in the view sync task, should we cancel polls for the certificate of that view ?

Reply:
`Timeout` is the start of view sync (if it's the second in a row) so probably shouldn't cancel any view sync related things